var/spack/repos/builtin/packages/gtk-doc/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class GtkDoc(AutotoolsPackage):
+    """GtkDoc is a tool used to extract API documentation from
+    C-code like Doxygen, but handles documentation of GObject
+    (including signals and properties) that makes it very
+    suitable for GTK+ apps and libraries. It uses docbook for
+    intermediate files and can produce html by default and
+    pdf/man-pages with some extra work."""
+
+    homepage = "https://wiki.gnome.org/DocumentationProject/GtkDoc"
```

Comment:
I would still have a `url` up here, as it is used by `spack versions` and `spack checksum`

Reply:
URL re-added